app/src/main/java/org/jellyfin/androidtv/ui/playback/PlaybackController.java

```diff
@@ -1282,6 +1284,13 @@ public void onEvent() {
                             // crossed fire off an async routine to update the program info
                             updateTvProgramInfo();
                         }
+
+                        // if no additional player errors occur within 30 seconds, reset the count
+                        if (playbackRetries > 0 && System.currentTimeMillis() - lastPlaybackError > 30000) {
+                            Timber.d("resetting playback errror count from %s to 0 because playback has become stable", playbackRetries);
+                            playbackRetries = 0;
+                        }
```

Comment:
Wouldn't it make more sense to do this check in the top of `playerErrorEncountered()` instead?

Reply:
absolutely. That makes way more sense